OlesiaBarannik/HW7/3.py

```diff
@@ -0,0 +1,12 @@
+# Task3. Write a function that calculates the number of characters
+# included in a given string
+# • input: "hello"
+# • output: {"h":1, "e":1,"l":2,"o":1}
+
```

Comment:
good job
but it is more optimized version of code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result